docs/installing-solidity.rst

```diff
@@ -9,11 +9,18 @@ Installing the Solidity Compiler
 Versioning
 ==========
 
-Solidity versions follow `semantic versioning <https://semver.org>`_ and in addition to
-releases, **nightly development builds** are also made available.  The nightly builds
-are not guaranteed to be working and despite best efforts they might contain undocumented
-and/or broken changes. We recommend using the latest release. Package installers below
-will use the latest release.
+Solidity versions mostly follow `Semantic Versioning <https://semver.org>`_.  We make
+an assurance in addition to SemVer that we will not introduce breaking changes into a patch
+level release. That means if you are using 0.x.y, then you can expect everything to work
+the same or better with 0.x.z with z > y.
+
+The nightly builds are not guaranteed to be working and despite best efforts they might
+contain undocumented and/or broken changes. We recommend using the latest release. Package
+installers below will use the latest release.
+
+When deploying contracts, you should use the latest released version of Solidity. This
```

Comment:
Now this paragraph reads a bit unmotivated :(

Reply:
This should be fixed with update above at f71b3de3d93598f534f15f660235c4bd3c78b5bd.

Yes?